src/actors/runtime/exitcode/vm_exitcodes.go

```diff
@@ -65,56 +64,30 @@ const (
 	InsufficientPledgeCollateral
 )
 
-type ExitCode struct {
-	Success          struct{}
-	SystemError      SystemErrorCode
-	UserDefinedError UserDefinedErrorCode
+func (x ExitCode) IsSuccess() bool {
+	return x == ExitCode(0)
 }
 
-func OK() ExitCode {
-	return ExitCode{
-		Success:          struct{}{},
-		SystemError:      0,
-		UserDefinedError: 0,
-	}
-}
-
-func SystemError(x SystemErrorCode) ExitCode {
-	return ExitCode{
-		Success:          struct{}{},
-		SystemError:      x,
-		UserDefinedError: 0,
-	}
-}
-
-func (x *ExitCode) IsSuccess() bool {
-	return x.Success == struct{}{} && x.SystemError == 0 && x.UserDefinedError == 0
+func (x ExitCode) IsError() bool {
+	return !x.IsSuccess()
 }
 
-func (x *ExitCode) IsError() bool {
-	return !x.IsSuccess()
+func (x ExitCode) Equals(e ExitCode) bool {
+	return x == e
 }
```

Comment:
This whole function is probably unnecessary now

Reply:
True, but it's slightly more convenient than `if exitCode == Exitcode(foo) {`. I'll remove it if you'd like.